keylime-agent/src/revocation.rs

```diff
@@ -155,26 +183,27 @@ pub(crate) fn run_action(
 /// # Arguments
 ///
 /// * `json` - The revocation message content
-/// * `secure_size` - The size of the secure mount
 /// * `config_actions` - Actions from the configuration file
 /// * `actions_dir` - Location of the pre-installed actions
-pub(crate) fn run_revocation_actions(
+fn run_revocation_actions(
     json: Value,
-    secure_size: &str,
-    config_actions: &str,
+    config_actions: Option<String>,
     actions_dir: &Path,
     allow_payload_actions: bool,
     work_dir: &Path,
     mount: &Path,
 ) -> Result<Vec<Output>> {
     // The actions from the configuration file takes precedence over the actions from the
     // actions_list file
-    let mut action_list = config_actions
+    let actions = match config_actions {
+        Some(actions) => actions,
+        None => "".to_string(),
+    };
```

Comment:
`.or_else(|| "".to_string())` ?

Reply:
Sorry, I do not get what you are suggesting here.

Reply:
Ah, maybe I am wrong: `actions = config_actions.unwrap_or_else(|| "".to_string())` ?

Reply:
Ah, yes, that makes sense.